content/docs/user-guide/how-to/best-practices.md

```diff
@@ -0,0 +1,74 @@
+# Best Practices for DVC Projects
+
+Data scientists, engineers, or managers may already know or can easily find
+answers to some of these questions. However, the variety of answers and
+approaches makes data science collaboration a nightmare. **A systematic approach
+is required.**
```

Comment:
This text doesn't seem to match the title.

> I have updated the doc and now its similar to QnA. The answers reflect what is the best way to achieve the solution to the question. This can be a useful doc for users. Thoughts on the format?

The contents are improving, thanks. But a QnA is not a guide. We had mentioned the idea of leaving a section for FAQs at the bottom of Best Practices doc, but not replace it completely.

I don't think this doc should be in QnA form. It's a how-to guide. See https://documentation.divio.com/how-to-guides/#how-to and reshape the doc that way.

If there are any topics left that can only or much better be expressed as question-and-answer, then let's create yet another separate page for that (hidden from nav for now).

Thanks

Reply:
QnA part was just for experiment. But I get the gist of how the best practices doc should look like (to the point solutions). I have  changed the starting para and also described each problem as a bullet point with a heading(heading could be improved more). Have added code blocks to some points. If this format looks fine then I can proceed to add more code blocks. Thanks.
Ps: the question in last part are same as those at start of this PR

Reply:
OK I reviewed the current format and left some comments.